packages/components/src/internal/components/ontology/OntologyTreePanel.tsx

```diff
@@ -60,24 +99,33 @@ export const OntologyTreePanel: FC<OntologyTreeProps> = props => {
                 }
             );
         },
-        [root]
+        //Needs to trigger for filters to ensure PathModels are loaded and update the FilterDialog values.
+        [root, filters]
     );
 
     const onSelect = (name: string, path: string, checked: boolean, isDirectory: boolean, node: PathNode): boolean => {
         if (node.active) onNodeSelection(node?.data);
         return true;
     };
 
+    const renderNodeHeader = props => {
+        return (
+        <Header {...props} showNodeIcon={false}>
+            {showFilterIcon && <FilterIcon {...props} filters={filters} onClick={onFilterChange} />}
+        </Header>);
+    };
+
     return (
         <FileTree
             loadData={loadData}
             onFileSelect={onSelect}
             allowMultiSelect={false}
-            showNodeIcon={false}
+            showNodeIcon={showFilterIcon}
```

Comment:
doesn't the `showNodeIcon` prop control the left side icon and the `showFilterIcon` is for the right side filter icon? I wasn't expecting that showFilterIcon would be used here for the value of showNodeIcon.